cli/block/interfaces.py

```diff
@@ -13,3 +13,17 @@ def get(self, identifier):
         Get a block by its identifier.
         """
         pass
+
+    def get_list(self, ids, head, limit, reverse):
+        """
+        Get a list of blocks.
+
+        A list of blocks could be filtered by blocks identifiers, head, limit, reverse.
+
+        Arguments:
+            ids (list, optional): identifiers to get a list of blocks by.
+            head (string, optional): block identifier to get a list of transactions from.
```

Comment:
It also, transactions from - transactions to.

Reply:
@Alladin9393, done.